docs/targets/index.md

```diff
@@ -10,20 +10,15 @@ The following is a list of Triggermesh event destinations known as `Targets`. So
     * [AWS SQS](./awssqs.md): Forward event payload to AWS Simple Queueing Service (SQS).
     * [AWS Kinesis](./awskinesis.md): Forward event payload to AWS Kinesis.
     * [AWS EventBridge](./awseventbridge.md): Forward arbitrary events to AWS EventBridge.
+* [Confluent](./confluent.md): Forward events to Confluent Kafka.
```

Comment:
Are we certain we want to show [this](https://docs.triggermesh.io/targets/confluent/) to the public?

Reply:
I am cleaning this up, and it will be updated in the next PR.

Reply:
Maybe this diff belongs in the "next PR" then?